tests/tests/lib/container.py

```diff
@@ -62,13 +61,25 @@ def cleanup(self):
             self.container.stop()
         self.container.remove()
 
-    def exec_run(self,command: (str | list[str]),raw_output: bool = False) -> Tuple[Optional[int],Union[Iterator[bytes],Any,Tuple[bytes,bytes]]]:
+    def exec_run(self, command: (str | list[str]), raw_output: bool = False) -> \
+            Tuple[Optional[int], Union[Iterator[bytes], Any, Tuple[bytes, bytes]]]:
+
         result, output = self.container.exec_run(command)
 
         if not raw_output and output:
             output = output.decode('utf-8').strip()
 
         return result, output
-    
-    def systemctl_daemon_reload(self) -> Tuple[Optional[int],Union[Iterator[bytes],Any,Tuple[bytes,bytes]]]:
+
+    def systemctl_daemon_reload(self) -> Tuple[Optional[int], Union[Iterator[bytes], Any, Tuple[bytes, bytes]]]:
         return self.exec_run("systemctl daemon-reload")
+
+    def wait_for_hirte(self):
+        result = 1
+        while result != 0:
```

Comment:
Indefinite loops are dangerous in tests, we need to have some timeout to fail the tests when it passes.
As an examle please take a look at https://github.com/oVirt/ovirt-system-tests/blob/master/ost_utils/assert_utils.py

Reply:
Definitely, and it is not a nice way to wait for hirte/-agent, but I couldn't think of sth better, unfortunately. 
I added `pytest-timeout` to the list of packages to mark each test individually with annotations - see [the proxy test](https://github.com/containers/hirte/pull/287/files#diff-1ccf81385cf06da6e891f723a10e0982a15d1d844d76a9c098e40aba6b0dc932R13) for example. I didn't add this to the other tests, yet.

Reply:
OK, so then let's add `TODO` here and let's fix in upcoming patches